subsys/caf/modules/power_manager.c

```diff
@@ -143,6 +147,19 @@ static void system_off_on_error(void)
 
 static void power_down(struct k_work *work)
 {
+	if (keep_alive_flag) {
+		keep_alive_flag = false;
+		power_down_counter_reset();
+		return;
+	}
+
+	power_down_interval_counter++;
+	if (power_down_interval_counter <
+	    (CONFIG_CAF_POWER_MANAGER_TIMEOUT) / (POWER_DOWN_CHECK_INTERVAL_SEC)) {
```

Comment:
please add  build assert somewhere to check if 
`POWER_DOWN_CHECK_INTERVAL_SEC < CONFIG_CAF_POWER_MANAGER_TIMEOUT`